packages/cli/src/reporters/abstract-list.ts

```diff
@@ -94,16 +108,20 @@ export default abstract class AbstractListReporter implements Reporter {
         status.push(sourceFile ? indentString(titleString, 2) : titleString)
       }
     }
+
     if (!opts.skipCheckCount) {
       status.push('')
       status.push([
+        counts.scheduling ? chalk.bold.blue(`${counts.scheduling} scheduling`) : undefined,
         counts.numFailed ? chalk.bold.red(`${counts.numFailed} failed`) : undefined,
         counts.numPassed ? chalk.bold.green(`${counts.numPassed} passed`) : undefined,
         counts.numPending ? chalk.bold.magenta(`${counts.numPending} pending`) : undefined,
```

Comment:
I think that we could change `pending` to `in-progress` or `running` now. It might make the difference between this category and `scheduling` more clear for users.